community/libwm/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+sed -i'' "s#/usr/local#/usr#" config.mk
+
+make
+make DESTDIR="$1" install
```

Comment:
This can just be:

```sh
make

install -D libwm.a "$1/usr/lib/libwm.a"
install -D wm.h    "$1/usr/include/wm.h"
```

We can then avoid patching the Makefile.

Reply:
or just `make DESTDIR="$1" PREFIX=/usr install` should be good too.